src/components/ebay-toggle-button-group/toggle-button-group.stories.ts

```diff
@@ -66,7 +86,10 @@ export default {
 
 export const Default = Template.bind({});
 Default.args = {
```

Comment:
The default should not have required column specifications. Those breakpoint-specific exceptions for devs are supposed to be overrides. 

<kbd><img width="1303" alt="image" src="https://github.com/eBay/ebayui-core/assets/1675667/12ed6248-1316-4cf3-bcf3-ef6b876c33c1"></kbd>

Reply:
These are simply for examples. We can remove them from the default example.